roles/matrix-bridge-mautrix-signal/defaults/main.yml

```diff
@@ -130,6 +130,10 @@ matrix_mautrix_signal_configuration_extension: "{{ matrix_mautrix_signal_configu
 # You most likely don't need to touch this variable. Instead, see `matrix_mautrix_signal_configuration_yaml`.
 matrix_mautrix_signal_configuration: "{{ matrix_mautrix_signal_configuration_yaml|from_yaml|combine(matrix_mautrix_signal_configuration_extension, recursive=True) }}"
 
+# Prevents the puppet from breaking when the signal security nuber changes.
+# The new security nuber will marked as trusted_unverified if this is set to true
+matrix_mautrix_signal_deamon_trust_new_security_nuber: false
```

Comment:
The name of the variable does not correspond to the configuration key that it influences (`SIGNALD_TRUST_NEW_KEYS`). This makes things harder for playbook users and is inconsistent with the rest of the playbook.

Reply:
The Bridge Bot called the Key 'new identity' when it displays a failure Message (only some times):
`⚠ Your message was not bridged: Identity failure occurred while sending message to +000. New identity: xxx`
The signal-daemon called it key and users of the original Signal App called the key 'safety number'. [Signal support article](https://support.signal.org/hc/en-us/articles/360007060632-What-is-a-safety-number-and-why-do-I-see-that-it-changed-)

Security Number was a translating mistake by me.
I would prefer to call it safety number, because that is the name which was known by signal users.

Or should i name it in the comment safety number and the variable `matrix_mautrix_signal_deamon_trust_new_keys`?

Reply:
If the configuration setting that we ultimately control is named `SIGNALD_TRUST_NEW_KEYS`, we should probably use a variable that has the same name (e.g. `matrix_mautrix_signal_deamon_trust_new_keys`).

The comment describing the variable can say "Security Number" and link to the Signal support article even.